integrationTests/buildSrc/build.gradle

```diff
@@ -73,6 +73,12 @@ subprojects {
         mustRunAfter project.tasks.check
       }
 
+      // FIXME this task dependency became obsolete with Gradle 8,
```

Comment:
> or whichever one it is

That's the hard part, and the documentation was not clear on the topic which tasks Gradle will consistently call for `buildSrc` projects.

I did not want to sink too much time into it. I would think we need to migrate this to a composite build, because this construct of "`buildSrc` project that when building the parent project has a task dependency on installation to a Maven repository such that the plugin is available in children projects" is too anomalous / off the beaten path.
So I wanted to make it stand out that this is indeed a workaround, and explicitly invoking the task from the GH action seemed a reliable way to fix it.